corehq/messaging/smsbackends/icds_nic/models.py

```diff
@@ -84,6 +88,12 @@ def get_generic_name(cls):
     def get_form_class(cls):
         return ICDSBackendForm
 
+    def destination_number_is_valid(self, phone_number):
+        """
+        phone_number is not expected to contain the leading + for this validation to work.
+        """
+        return phone_number.startswith(INDIA_COUNTRY_CODE) and phone_number != INDIA_COUNTRY_CODE
```

Comment:
not sure, i kind of like the readability this way. the main issue is that there are a lot of numbers that aren't filled in and only consist of just the country code, so i wanted to be explicit that we're ignoring those